OOPBASICSPART1TASK6/src/com/gyk/java/oop/basics/TravelDestination.java

```diff
@@ -0,0 +1,53 @@
+package com.gyk.java.oop.basics;
+
+public class TravelDestination {
+    private String name;
+    private String city;
+    private int distance;
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        TravelDestination that = (TravelDestination) o;
+        return distance == that.distance && name.equals(that.name) && city.equals(that.city);
+    }
+
+    @Override
+    public int hashCode() {
+        return 0;

```

Comment:
it should return a hash of the included class values. in most cases IDE generated method is fine

Reply:
ok